Modules/Devices/test/Devices.Application.Tests/Tests/PushNotifications/DatawalletModified/HandlerTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Backbone.BuildingBlocks.Application.PushNotifications;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Application.DomainEvents.Incoming.DatawalletModificationCreated;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications;
+using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications.Handles;
+using Backbone.Modules.Devices.Domain.DomainEvents.Incoming.DatawalletModificationCreated;
+using Backbone.Modules.Devices.Infrastructure.PushNotifications;
+using Backbone.Modules.Devices.Infrastructure.PushNotifications.Connectors;
+using Backbone.UnitTestTools.BaseClasses;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.PushNotifications.DatawalletModified;
+
+public class HandlerTests : AbstractTestsBase
```

Comment:
Sorry, I should have been more clear: I was referring to a test for the PushService's new Send method.
The test you wrote isn't ideal because of the 2-layer-mocks/-fakes you built. That makes it way more complicated than it has to be. If you would test the PushService directly, you would save one layer of fakes. 

Also, instead of 
```c#
A.CallTo(() => fakeConnector.Send(A<IEnumerable<PnsRegistration>>._, A<IPushNotification>._))
            .Invokes((IEnumerable<PnsRegistration> registrations, IPushNotification _) => notifiedDevices.AddRange(registrations));
...
notifiedDevices.Should().HaveCount(1);
notifiedDevices.First().Should().Be(deviceB);
```

you should do:
```c#
A.CallTo(() => mockConnector.Send(...)).MustHaveHappendOnceExactly()
```

You can keep this test though. Just simplify it and only assert that `IPushService.Send` is called.